client/state/site-settings/exporter/actions.js

```diff
@@ -96,7 +96,7 @@ export function advancedSettingsFail( siteId, error ) {
  * @param  {Number}   siteId  The ID of the site to export
  * @return {Function}         Action thunk
  */
-export function startExport( siteId, { exportAll = true } = {} ) {
+export function startExport( siteId, { exportAll = true } = {}, siteType = 'wpcom' ) {
```

Comment:
The jsdoc looks outdated here. The second argument looks like an options argument, which we might want to add to.

In this case we can figure out site type from the siteId, though it is weird to do so in an action. I would maybe suggest splitting this out to two different actions if we do need to call a different endpoint.